.github/workflows/e2e.yml

```diff
@@ -9,6 +9,22 @@ jobs:
     name: E2E
     timeout-minutes: 30
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        deploytool: ['operator']
+        globalnet: ['', 'globalnet']
+        cable_driver: ['libreswan', 'wireguard', 'vxlan']
+        ovn: ['', 'ovn']
+        k8s_version: ['1.20.2']
+        exclude:
+          - ovn: 'ovn'
+            globalnet: 'globalnet'
+          - ovn: 'ovn'
+            cable_driver: 'wireguard'
+        include:
+          # This is the oldest K8s version we try to support
+          - k8s_version: 1.17.17
```

Comment:
Please note that your changes should now go in: 
https://github.com/submariner-io/submariner/blob/devel/.github/workflows/e2e-full.yml

As those tests are only activated when the ready-to-test label is assigned due to enough reviews.
```suggestion
```